onecodex/notebooks/exporters.py

```diff
@@ -209,12 +210,14 @@ def from_notebook_node(self, nb, resources=None, **kw):
 
         try:
             document_id = _upload_document_fileobj(
-                BytesIO(output), file_name, ocx._client.session, ocx.Documents._resource
+                BytesIO(output), file_name, ocx.Documents._resource
             )
         except UploadException as exc:
             resp = json.dumps({"status": 500, "message": str(exc)})
             return resp, resources
-        except Exception:
+        except Exception as exc:
+            client = get_raven_client()
```

Comment:
Do we need to do this anymore? I _think_ in the newest versions of the Sentry library you can just do `sentry_sdk.capture_exception` but maybe we need to upgrade.